src/Behat/Symfony2Extension/Context/Argument/ServiceArgumentResolver.php

```diff
@@ -60,10 +60,6 @@ public function resolveArguments(ReflectionClass $classReflection, array $argume
      */
     private function resolveArgument($argument)
     {
-        if (is_array($argument)) {
-            return array_map(array($this, 'resolveArgument'), $argument);
```

Comment:
this does not resolve argument recursively for arrays, which look wrong to me, as it does not allow services inside an array anymore